app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt

```diff
@@ -132,7 +132,7 @@ class StoryActivityTest {
   }
 
   private fun setUpTestApplicationComponent() {
-    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(storyActivityTest = this)
```

Comment:
We dont need named argument for this line.

Reply:
@jkugsiya Please reply to this comment.

Reply:
https://github.com/oppia/oppia-android/commit/8ba9f9b8e206dc434c26d0e03a846bc8081e1a09
This requested change is provided in the above linked commit!